front/src/common/BootstrapSNCF/ModalSNCF/ModalProvider.js

```diff
@@ -60,7 +69,7 @@ function ModalSNCF() {
       }
     };
     document.addEventListener('mousedown', handleClickOutside);
-    return () => {
+    return function cleanup() {
```

Comment:
Why did you add function cleanup here instead of () => ?

Reply:
I thought it was more explicit..? Anyway it's reduced to transpilation so might as well leave it readable?